index.md

```diff
@@ -7,4 +7,29 @@ position: 1
 tags: overview, welcome
 ---
 
-This is a placeholder document for the index page.
\ No newline at end of file
+The {{site.tp}} Knowledge Base portal gives answers to common user inquiries or issues, identified by the engineering team in the form of How-To and Troubleshooting articles. The easiest way to find relevant information is to search for key words in the *Search Knowledge Base* box at the top right corner.
+
+If you don't find information about the exact issue/scenario here, continue with the next steps in order to utilize the full potential of our support service:
+
+# **Step 1:** Browse the support resources
+The {{site.tp}} support system provides a number of **[support resources](http://docs.telerik.com/platform/appbuilder/troubleshooting/support/support-resources)** which you can browse for solutions to your problem. You can apply the suggested workarounds and continue your work without submitting a support request.
+
+# **Step 2:** Submit a support request
+
+If you cannot find a solution to your problem in the support resources, you need to submit a support request. Your current subscription plan determines the level of support assistance that you can receive. For more information about subscription plans, see <a href="http://www.telerik.com/purchase/platform" target="_blank">Buy Telerik Platform</a>.
+
+## Ask for assistance in the forums
+
+In the forums, you can ask for assistance for technical, developmental and general issues. When you **[submit a support request in the forums](http://docs.telerik.com/platform/appbuilder/troubleshooting/support/community-support)**, you can receive feedback from other community members. The {{site.tp}} support team might also assist you.
+
+Post your issue only once in the most relevant section. Make sure to provide a detailed description of the issue, including steps to reproduce, stack traces, code snippets and logs, where applicable.
+
+## Submit a support ticket
+
+If your subscription plan covers personal support, you can **[submit a support ticket](http://docs.telerik.com/platform/appbuilder/troubleshooting/support/support-tickets)**.
+
+You can follow the workflow on that page to ensure that you have provided the information that the support team needs. Providing complete and accurate information in your support ticket might significantly speed up the response process.
```

Comment:
Instead of focusing on what "might happen", I think it's better to focus on what "will". For instance, we can say: 
Providing complete and accurate information in your support ticket will decrease its resolution time.